internal/controller/stackrunjob_controller.go

```diff
@@ -42,6 +43,7 @@ import (
 )
 
 const jobSelector = "stackrun.deployments.plural.sh"
+const jobTimout = time.Minute * 10
```

Comment:
Think this should be 30 or 60 minutes (could have a weird scheduling edge case that takes 10 minutes for benign reasons I think)

Reply:
done